src/aap_eda/settings/default.py

```diff
@@ -11,823 +11,39 @@
-    else:
-        params |= {
-            "HOST": REDIS_HOST,
-            "PORT": REDIS_PORT,
-        }
-        if REDIS_TLS:
-            params["SSL"] = True
-        else:
-            # TODO: Deprecate implicit setting based on cert path in favor of
-            #       MQ_TLS as the determinant.
-            if REDIS_CLIENT_CERT_PATH and REDIS_TLS is None:
-                params["SSL"] = True
-            else:
-                params["SSL"] = False
-    return params
-
-
-def _rq_redis_client_additional_parameters():
-    params = {}
-    if (not REDIS_UNIX_SOCKET_PATH) and REDIS_CLIENT_CERT_PATH:
-        params |= {
-            "ssl_certfile": REDIS_CLIENT_CERT_PATH,
-            "ssl_keyfile": REDIS_CLIENT_KEY_PATH,
-            "ssl_ca_certs": REDIS_CLIENT_CACERT_PATH,
-        }
-    return params
-
-
-def rq_standalone_redis_client_instantiation_parameters():
-    params = _rq_common_parameters() | _rq_redis_client_additional_parameters()
-
-    # Convert to lowercase for use in instantiating a redis client.
-    params = {k.lower(): v for (k, v) in params.items()}
-    return params
-
-
-def rq_redis_client_instantiation_parameters():
-    params = rq_standalone_redis_client_instantiation_parameters()
-
-    # Include the HA cluster parameters.
-    if REDIS_HA_CLUSTER_HOSTS:
-        params["mode"] = "cluster"
-        params["redis_hosts"] = REDIS_HA_CLUSTER_HOSTS
-        params["socket_keepalive"] = _get_boolean("MQ_SOCKET_KEEP_ALIVE", True)
-        params["socket_connect_timeout"] = settings.get(
-            "MQ_SOCKET_CONNECT_TIMEOUT", 10
-        )
-        params["socket_timeout"] = settings.get("MQ_SOCKET_TIMEOUT", 150)
-        params["cluster_error_retry_attempts"] = settings.get(
-            "MQ_CLUSTER_ERROR_RETRY_ATTEMPTS", 3
-        )
-        from redis.backoff import ConstantBackoff
-        from redis.retry import Retry
-
-        params["retry"] = Retry(backoff=ConstantBackoff(3), retries=20)
-    return params
-
-
-# A list of queues to be used in multinode mode
-# If the list is empty, use the default singlenode queue name
-RULEBOOK_WORKER_QUEUES = settings.get("RULEBOOK_WORKER_QUEUES", [])
-if isinstance(RULEBOOK_WORKER_QUEUES, str):
-    RULEBOOK_WORKER_QUEUES = RULEBOOK_WORKER_QUEUES.split(",")
-
-if len(set(RULEBOOK_WORKER_QUEUES)) != len(RULEBOOK_WORKER_QUEUES):
-    raise ImproperlyConfigured(
-        "The RULEBOOK_WORKER_QUEUES setting must not contain duplicates."
-    )
-
-# If the list is empty, use the default queue name for single node mode
-if not RULEBOOK_WORKER_QUEUES:
-    RULEBOOK_WORKER_QUEUES = ["activation"]
-
-DEFAULT_QUEUE_TIMEOUT = settings.get("DEFAULT_QUEUE_TIMEOUT", 300)
-DEFAULT_RULEBOOK_QUEUE_TIMEOUT = settings.get(
-    "DEFAULT_RULEBOOK_QUEUE_TIMEOUT", 120
+from ansible_base.lib import dynamic_config
+from ansible_base.lib.dynamic_config import (
+    export,
+    factory,
+    load_envvars,
+    load_standard_settings_files,
 )
+from split_settings.tools import include
```

Comment:
We have to remove the usage of split_settings and replace with dynaconf equivalent, that is bacause when Dynaconf loads it is tracked on the inspection history.